src/com/google/enterprise/gsafeed/GsaFeedFileSender.java

```diff
@@ -34,8 +34,8 @@
 
 /** Takes an XML feed file for the GSA, sends it to GSA and
   then reads reply from GSA. */
-/* Modified from GsaFeedFileSender in adaptor library. */
-class GsaFeedFileSender {
+// Modified from GsaFeedFileSender in adaptor library.
+public class GsaFeedFileSender {
```

Comment:
why do methods in this class have to become public at this time?

Reply:
I wanted to use them in an example. It's possible as we get further along that we'll end up hiding this again in some different interface, but for now I wanted to have something running to verify that this was working outside the adaptor library.